

# [fail] [AptaMotif](#AptaMotif)

J. Hoinka, E. Zotenko, A. Friedman, Z.E. Sauna, T.M. Przytycka, Identifification of sequence-structure RNA binding motifs for SELEX-derived aptamers, Bioinformatics 28 (2012) i215ei223.

https://doi.org/10.1093/bioinformatics/bts210

## ribosomal protein L22

Dobbelstein M.,  Shenk T.. {In vitroselection of RNA ligands for the ribosomal L22 protein associated with Epstein-Barr virus-expressed RNA by using randomized and cDNA-derived RNA libraries}, J. Virol., 1995, vol. 69 (pg. 8027-8034)

https://journals.asm.org/doi/abs/10.1128/jvi.69.12.8027-8034.1995

The RNA library consists of a pool of 16 aptamers containing a 30 nt, initially randomized region flanked by primer binding sites of length 24 nt (5′) and 23 nt (3′), respectively. Dobbelstein and Shenk identified a highly conserved stem–loop structure as the functional site of the L22 binding aptamers. The hairpin consists of 6–9 nts, with the most 3′ nt being thymine in most of the species and a (5′)G−C(3′) configuration forming the closing base pair. This scenario represents a motif of intermediate complexity since the motif is continuous in primary structure but it contains a large number of insertions, deletions and variations in sequence.

## amino acid isoleucine

Lozupone C., et al. Selection of the simplest RNA that binds isoleucine, RNA, 2003, vol. 9 (pg. 1315-1322)

https://rnajournal.cshlp.org/content/9/11/1315.short

# [1] [AptaCluster](#AptaCluster)

J. Hoinka, A. Berezhnoy, Z.E. Sauna, E. Gilboa, T.M. Przytycka, AptaCluster - a method to cluster HT-SELEX aptamer pools and lessons from its application, Res Comput Mol Biol 8394 (2014) 115e128.

https://link.springer.com/chapter/10.1007/978-3-319-05269-4_9

## Interleukin 10 receptor alpha chain (IL-10RA) 

### https://www.ebi.ac.uk/ena/browser/view/PRJNA315881

https://www.ncbi.nlm.nih.gov/bioproject/315881

We applied 4 rounds of selection and cDNA generated from round 5 bound fractions as well as RNA recovered from bound fractions at rounds 2, 3 and 4 was amplified and sequenced using Illuminas HiSeq 2500 device with 100-cycle pairedend sequencing protocol (see HT-SELEX Experiment section for the experimental protocol).

For the entire experiment, a total of 12895554 sequences where retrieved of which 4621438 species belonged to round 5, 1923823 to round 4, 2181720 to round 3, and 4168573 to round 2. Out of these respectively 617220, 1021668, 1902904, and 3857210 were unique.

# [fail-1] [Mpbind](#Mpbind)

P. Jiang, S. Meyer, Z. Hou, N.E. Propson, H.T. Soh, J.A. Thomson, R. Stewart, Mpbind, A Meta-motif-based statistical framework and pipeline to Predict Binding potential of SELEX-derived aptamers, Bioinformatics 30 (2014) 2665e2667.

https://doi.org/10.1093/bioinformatics/btu348

##  human embryonic stem cell

### https://www.morgridge.net/dmz/www-dmz/upload/files/peng/Aptamer/

![image-20221103192109429](SELEX-DATASET/image-20221103192109429.png)

## ESRP1

we tested it on another SELEX-Seq dataset (ESRP1) (Dittmar et al., 2012). Dittmar et al. generated five rounds of SELEX-Seq (ESRP1)

data (R0, R2, R3, R6 and R7).

Dittmar,K.A. et al. (2012) Genome-wide determination of a broad ESRP-regulated posttranscriptional network by high-throughput sequencing. Mol. Cell. Biol., 32, 1468–1482.

https://doi.org/10.1128/MCB.06536-11

# [2] FasterAptamer

K.K. Alam, J.L. Chang, D.H. Burke, FASTAptamer: a bioinformatic toolkit for high-throughput sequence analysis of combinatorial selections, Mol. Ther. Nucleic Acids 4 (2015), e230.

https://doi.org/10.1038/mtna.2015.4

The utility of the FASTAptamer toolkit was evaluated by analyzing two populations of RNA aptamers selected to bind HIV-1 Reverse Transcriptase (RT). The first population (70HRT14) was selected through 14 cycles of selection by nitrocellulose filter retention and native gel shift; this population has been described previously and thus served as a control for the analysis. The 70HRT14 population was subjected to one additional cycle of selection under slightly modified conditions to generate the second population (70HRT15) and both populations underwent HTS.

## HIV-1 Reverse Transcriptase

For the present study, a total of 2,160,216 sequences were obtained for 70HRT14 and 1,987,867 for 70HRT15 after pre-processing.

https://burkelab.missouri.edu/fastaptamer.html

### $70HRT_{14}$

https://burkelab.missouri.edu/fastaptamer/sample-data/70HRT14.fastq.zip

### $70HRT_{15}$

https://burkelab.missouri.edu/fastaptamer/sample-data/70HRT15.fastq.zip



# [ok] [AptaMut](#AptaMut)

J. Hoinka, A. Berezhnoy, P. Dao, Z.E. Sauna, E. Gilboa, T.M. Przytycka, Large scale analysis of the mutational landscape in HT-SELEX improves aptamer discovery, Nucleic Acids Res. 43 (2015) 5699e5707.

https://doi.org/10.1093/nar/gkv308

A DNA template for the selection library was ordered from IDT (Coralville, IA). 1 nM of each N40 template (5′-TCTCGATCTCAGCGAGTCGTCG-N40-CCCATCCCTCTTCCTCTCTCCC-3′) and 5′ primer (5′-GGGGGAATTCTAATACGACTCACTATAGGGAGAGAGGAAGAGGGATGGG-3′) were annealed together, extended with Taq polymerase (Life Science) and transcribed *in vitro* using Durascribe (in-vitro transcription) IVT kit (Illumina). The random R0 RNA was purified by denaturing PAGE and, after preclearing with human IgG-coated (Sigma) beads (GE Healthcare), used for in-vitro selection. 1 nM of R0 RNA was used in a first round of selection to coincubate with 0.3 nM of bead-bound human IL-10RA-Fc fusion protein (Novus Biologicals) in 100 mM NaCl selection buffer. After washes, a recovered bound RNA fraction was reverse transcribed using the cloned AMV RT kit (Life Science). cDNA was amplified by either emulsion or open PCR using Platinum Taq PCR kit (Life Science) as described below. The DNA template was used to IVT RNA for the next round. During subsequent rounds, amount of protein was reduced 25\% each time, while concentration of NaCl was gradually increased to 150 mM.

##  IL-10RA

see [AptaCluster](#AptaCluster)

https://www.ebi.ac.uk/ena/browser/view/PRJNA315881

https://www.ncbi.nlm.nih.gov/bioproject/315881

# [1] [AptaTRACE](#AptaTRACE)

P. Dao, J. Hoinka, M. Takahashi, J. Zhou, M. Ho, Y. Wang, F. Costa, J.J. Rossi, R. Backofen, J. Burnett, T.M. Przytycka, AptaTRACE elucidates RNA sequence structure motifs from selection trends in HT-SELEX experiments, Cell Syst 3 (2016) 62e70.

https://doi.org/10.1016/j.cels.2016.07.003

## simulated data

AptaSim

## C-C chemokine receptor type 7 (CCR7)

### https://www.ncbi.nlm.nih.gov/bioproject/321551

we show our results of applying AptaTRACE to an in vitro selection consisting of high-throughput data from nine rounds of cell-SELEX ([Takahashi et al., BioProject: PRJNA321551](http://www.ncbi.nlm.nih.gov/bioproject/321551)).

we applied AptaTRACE to the results of an in vitro cell-SELEX experiment targeting the C-C chemokine receptor type 7 (CCR7), where the initial pool as well as seven of nine selection rounds have been sequenced, averaging 40 million aptamers per cycle

# [fail-1] APTANI

Caroli J. et al.  (2016) APTANI: a computational tool to select aptamers through sequence-structure motif analysis of HT-SELEX data. Bioinformatics, 32, 161–164.

https://doi.org/10.1093/bioinformatics/btv545

## murine IL4R$\alpha$



Roth,F. et al. (2012) Aptamer-mediated blockade of IL4Ra triggers apoptosis of MDSCs and limits tumor progression. Cancer Res., 72, 1373–1383.

https://doi.org/10.1158/0008-5472.can-11-2772

# [None] AptaPLEX

J. Hoinka, T. Przytycka, AptaPLEX - a dedicated, multithreaded demultiplexer for HT-SELEX data, Methods 106 (2016) 82e85.

https://doi.org/10.1016/j.ymeth.2016.04.011

# [None] AptaSUITE

J. Hoinka, R. Backofen, T.M. Przytycka, AptaSUITE: a full-featured bioinfor matics framework for the comprehensive analysis of aptamers from HT-SELEX experiments, Mol. Ther. Nucleic Acids 11 (2018) 515e517.

https://doi.org/10.1016/j.omtn.2018.04.006

# [fail-2] SMART-Aptamer

J. Song, Y. Zheng, M. Huang, L. Wu, W. Wang, Z. Zhu, Y. Song, C. Yang, A sequential multidimensional analysis algorithm for aptamer identifification based on structure analysis and machine learning, Anal. Chem. 92 (2020) 3307e3314.

https://doi.org/10.1021/acs.analchem.9b05203

##  human embryonic stem cells (hESCs)

see [Mpbind](#Mpbind)

## epithelial cell adhesion molecule (EpCAM)

Song Y, Zhu Z, An Y, et al. Selection of DNA aptamers against epithelial cell adhesion molecule for cancer cell imaging and circulating tumor cell capture[J]. Analytical chemistry, 2013, 85(8): 4141-4149.

https://doi.org/10.1021/ac400366b

The HTS SELEX data of EpCAM that support the findings of this study are available from the corresponding authors on reasonable request

## cell-surface vimentin (CSV)

Satelli A, Mitra A, Brownlee Z, et al. Epithelial–Mesenchymal Transitioned Circulating Tumor Cells Capture for Detecting Tumor ProgressionDetection of EMT CTC[J]. Clinical Cancer Research, 2015, 21(4): 899-906.

https://doi.org/10.1158/1078-0432.CCR-14-0894

# [2] RaptRanker {#RaptRanker}

R. Ishida, T. Adachi, A. Yokota, H. Yoshihara, K. Aoki, Y. Nakamura, M. Hamada, RaptRanker: in silico RNA aptamer selection from HT-SELEX experiment based on local sequence and structure information, Nucleic Acids Res. 48 (2020) e82.

https://doi.org/10.1093/nar/gkaa484

## human recombinant transglutaminase 2 (TG2; Cat. No. 4376-TG) 

### https://www.ebi.ac.uk/ena/browser/view/PRJDB9110

DRA009383 

![image-20221103203932631](SELEX-DATASET/image-20221103203932631.png)

## human recombinant integrin alpha V beta 3 (αVβ3; Cat. No. 3050-AV)

### https://www.ebi.ac.uk/ena/browser/view/PRJDB9111

DRA009384

![image-20221103203946399](SELEX-DATASET/image-20221103203946399.png)

# [fail] AptCompare

K.R. Shieh, C. Kratschmer, K.E. Maier, J.M. Greally, M. Levy, A. Golden, AptCompare: optimized de novo motif discovery of RNA aptamers via HTS-SELEX, Bioinformatics 36 (2020) 2905e2906.

https://doi.org/10.1093/bioinformatics/btaa054

## human transferrin receptor (htfr)

The transferrin receptor is an integral transmembrane protein that plays a role in iron uptake by cells. Increased expression of the receptor has been seen in cells with increased rates of proliferation. Thus, the transferrin receptor has been used to target cancer cells. We had previously identified three aptamers to the hTfR from five rounds of conventional, non-HTS SELEX using an N50 library consisting of sequences with a random region of length 50 (Wilner et al., 2012). The first four rounds of selection were performed on a recombinant receptor and the fifth round was selected on HeLa cells, an immortalized cell line with high expression of the hTfR. Three of the thirteen clones, named c2, c4, and c11 (Fig. S1), were found to be functional binders. This selection was extended, resulting in the identification of three additional aptamers (Maier et al., 2016). The initial, randomized pool and the five selection rounds were sequenced on an Illumina MiSeq platform (San Diego, CA, USA) using single-end sequencing at the Epigenomics Sequencing Center at the Albert Einstein College of Medicine. The raw FASTQ files were used for the AptCompare analysis.

Maier K E, Jangra R K, Shieh K R, et al. A new transferrin receptor aptamer inhibits new world hemorrhagic fever mammarenavirus entry[J]. Molecular Therapy-Nucleic Acids, 2016, 5: e321.

https://doi.org/10.1038/mtna.2016.32

# [fail-1] APTANI2

Caroli,J. et al. (2016) APTANI: a computational tool to select aptamers through sequence-structure motif analysis of HT-SELEX data. Bioinformatics, 32, 161–164.

https://doi.org/10.1093/bioinformatics/btz897

## Human prostate cancer cell lines

Speransky,S. et al. (2019) A novel RNA aptamer identififies plasma membrane ATP synthase beta subunit as an early marker and

therapeutic target in aggressive cancer. Breast Cancer Res. Treat., 176, 271–289.

https://link.springer.com/article/10.1007/s10549-019-05174-3

## C-C chemokine receptor type 7 (CCR7)

The PRJNA321551 data set consists of 10 different Cell-SELEX cycles (cycle 0 and 9 subsequent rounds of selection) performed using an RNA library of aptamers, with a variable region of 30 nucleotides, targeting CCR7 (Dao *et al.*, 2016). In their experiment, the authors used AptaTRACE to select 5 most representative secondary structure motifs (i.e., CTGTG, TTATT, GTTTA, ATGTT and GTGTC). Here, we applied APTANI2 and APTANI on the data from cycle 9 (SRX1763618), using default settings and a frequency threshold of 10-8, to verify if the two methods could identify the secondary structure motifs selected, as the most significant, by AptaTRACE. Among the 564,257 retrieved motifs, APTANI2 ranked the secondary structure motifs of (Dao *et al.*, 2016) in the top 99th percentile of the MtfScore ranked motif list. Conversely, APTANI did not identified any of the 5 structural motifs. The fact that APTANI2 ranks in the top positions the motifs selected by AptaTRACE supports the biological relevance of these 5 secondary structures and the efficacy of APTANI2, as compared to APTANI and AptaTRACE.

see [AptaTRACE](#AptaTRACE)

https://www.ebi.ac.uk/ena/browser/view/PRJNA321551

## IL-10RA

The PRJNA315881 data set comprises 5 subsequent rounds of selection against the human IL-10RA-Fc fusion protein, using an RNA library of aptamers with fixed flanking regions surrounding a variable region of 40 nucleotides (Levay *et al.*, 2015). Using an **ad-hoc** HT-SELEX experimental design and sequence clustering, Levay and coworkers identified 33 sequences as potential binders based on high prevalence and enrichment rate values. In a binding assay, 7 of these sequences displayed high affinity to human IL-10RA (Kd<20 nM) and 4 of them were further proven to be target specific using human IgG and murine IL-10RA as control references (Table S3). Here, we used APTANI2 and APTANI on the data from cycle 5 (SRX1653575) to verify if our tools were able to indicate these 7 aptamers as potential binders. APTANI2 ranked all 7 sequences in the top 99% of the ranked list (Table S3), while APTANI could not analyze the 53 GB of cycle 5 data due to its intrinsic limitations in memory usage and computational time.

see [AptaMut](#AptaMut)

https://www.ebi.ac.uk/ena/browser/view/PRJNA315881

# [ok] FSBC

Kato, S., Ono, T., Minagawa, H. *et al.* FSBC: fast string-based clustering for HT-SELEX data. *BMC Bioinformatics* **21**, 263 (2020).

https://doi.org/10.1186/s12859-020-03607-1

##  human embryonic stem cells

see [Mpbind](#Mpbind)

# [1] MLPD

A. Bashir, Q. Yang, J. Wang, S. Hoyer, W. Chou, C. McLean, G. Davis, Q. Gong, Z. Armstrong, J. Jang, et al., Machine learning guided aptamer refifinement and discovery, Nat. Commun. 12 (2021) 2366.

https://doi.org/10.1038/s41467-021-22555-9

## NGAL (R&D systems, C-terminal His-tag)

### https://www.ebi.ac.uk/ena/browser/view/PRJNA672779

raw sequencing reads are available in the NCBI SRA repository, PRJNA672779



# [ok] RaptGen

Iwano, N., Adachi, T., Aoki, K. *et al.* Generative aptamer discovery using RaptGen. *Nat Comput Sci* **2**, 378–386 (2022). 

https://doi.org/10.1038/s43588-022-00249-6

![image-20221103205305939](SELEX-DATASET/image-20221103205305939.png)

## human transglutaminase 2

see [RaptRanker](#RaptRanker)

 Dataset A, targeting human transglutaminase 2, consists of nine SELEX rounds from 0 to 8

## human integrin alpha V beta 3 

see [RaptRanker](#RaptRanker)

Dataset B, targeting human integrin alpha V beta 3, consists of four rounds from 3 to 6. 

 



